tools.py

```diff
@@ -19,11 +19,33 @@ def MSE(yhat, y):
     mse : float
         The mean squared error between yhat and y.
     '''
-    mse = np.sqrt(np.mean((yhat.flatten() - y.flatten())**2))
+    mse = np.sqrt(np.mean((y.flatten() - yhat.flatten())**2))
```

Comment:
is y.flatten() always greater than yhat.flatten()?
You could add a np.absolute() if not.

Reply:
It doesn't matter for this one because it is squared

Reply:
oh yes, you are right